test/common/grpc/grpc_web_filter_test.cc

```diff
@@ -120,19 +117,32 @@ TEST_F(GrpcWebFilterTest, TextUnary) {
   // Tests response headers.
   Http::TestHeaderMapImpl response_headers;
   response_headers.addViaCopy(Http::Headers::get().Status, "200");
-  response_headers.addViaCopy(Http::Headers::get().ContentType,
```

Comment:
It's not required at this moment, as the downstream content-type is gRPC-Web instead of gRPC. I plan to create some pull requests to add more error handling, may include additional check on the backend response content-type.